test/plugin/base_test.rb

```diff
@@ -305,20 +305,83 @@ def test_structured_payload_malformatted_log
   def test_structured_payload_json_log
     setup_gce_metadata_stubs
     setup_logging_stubs do
-      d = create_driver
-      json_string = '{"msg": "test log entry 0", "tag2": "test", "data": 5000}'
+      d = create_driver(APPLICATION_DEFAULT_CONFIG)
+      json_string = '{"msg": "test log entry 0", "tag2": "test", ' \
+                    '"data": 5000, "some_null_field": null}'
+      d.emit('message' => 'notJSON ' + json_string)
```

Comment:
I think the test is missing for the condition that asserts the existence of one field only

Reply:
@crassirostris Can you please clarify? We expect the whole JSON to end up in `textPayload`.